CONTRIBUTING.md

```diff
@@ -1,3 +1,101 @@
 # Contributing Guidelines
 
-TODO.
+Thanks for your interest in contributing to the Creative Labs website, aka Sunshine! ❤️
+
+These guidelines describe how to set up your environment and submit your own contributions! Please read it 
+carefully and, if there is anything that isn't up-to-date, feel free to submit a PR with your corrections 🙃
+
+- [Getting Started](#Getting-Started)
+- [Pull Requests](#Pull-Requests)
+  - [Step 1: Open an Issue](#Step-1:-Open-an-Issue)
+  - [Step 2: Design](#Step-2:-Design)
+  - [Step 3: Beep Boop](#Step-3:-Beep-Boop)
+  - [Step 4: Commit](#Step-4:-Commit)
+  - [Step 5: Pull Request](#Step-5:-Pull-Request)
+  - [Step 6: Merge](#Step-6:-Merge)
+
+## Getting Started
+
+We use [`yarn`](https://classic.yarnpkg.com/en/docs/install#mac-stable) as our package manager.
+
+The basic commands to get this repository and start are:
+
+```
+$ git clone https://github.com/UCLA-Creative-Labs/sunshine.git
+$ cd sunshine
+$ yarn install
+```
+
+To expedite local development, run locally:
+
+```
+$ yarn install
```

Comment:
If this is for running the website on a local development server, this should be `yarn start`.
```suggestion
$ yarn start
```

Reply:
lmao nice catch.. twas late